### PR TITLE
feat(Songsterr): add presence

### PR DESCRIPTION
**Pull request by AlperShal**
## Description 
I added support for songsterr.com, a website for searching instruments' tabs. It's one of the most popular ones.

## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

Uwl0Pzt

omdI5Rr

</details>

### CODE REVIEW

File: websites/S/Songsterr/metadata.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	"description": {
+		"en": "Songsterr is one of the most popular sites to find and play guitar or any instrument's tabs.",
+		"tr": "Songsterr gitar ya da herhangi bir çalgının tablarını bulmak ya da çalmak için en popüler sitelerden birisi."
+	},
+	"url": [
+		"www.songsterr.com",
+		"songsterr.com"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/MXQP0Lq.png",
```
Comment (theusaf) [inline websites/S/Songsterr/metadata.json:17]: [Logos](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo) are required to have a resolution of 512x512px

**Review by theusaf — CHANGES_REQUESTED** [review]
Make sure to resolve CI issues

**Comment by AlperShal** [comment]
Sorry to take your time about the lint failing. I really misunderstood how it works. Also thought the logo was required to have _at least_ 512x512 resolution, not exactly. Sorry about that too. Should be fine now.

File: websites/S/Songsterr/presence.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
+const presence = new Presence({
+	clientId: "1112463096368353300",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/MXQP0Lq.png",
```
Comment (theusaf) [inline websites/S/Songsterr/presence.ts:7]:
```suggestion
		largeImageKey: "https://i.imgur.com/FbLdpEW.png",
```

**Review by theusaf — CHANGES_REQUESTED** [review]

**Comment by theusaf** [comment]
> Sorry to take your time about the lint failing. I really misunderstood how it works. Also thought the logo was required to have _at least_ 512x512 resolution, not exactly. Sorry about that too. Should be fine now.

It's fine. The logo being required to be 512x512 was a relatively recent change, and the documentation on the website vs here is a bit out of sync, which may be adding a bit of confusion.

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by theusaf — APPROVED** [review]

**Pull request merged**